test/e2e/ingress.go

```diff
@@ -1027,3 +1027,129 @@ var _______ = describe("Ingress tests simple NLB", func() {
 		Expect(resp.Header.Get("Request-Host")).To(Equal(hostName))
 	})
 })
+
+var ________ = describe("Ingress tests for OPA filters", func() {
+	f := framework.NewDefaultFramework("skipper-ingress-with-opa")
+	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelBaseline
+	var (
+		cs  kubernetes.Interface
+		jig *ingress.TestJig
+	)
+
+	It("Should activate OPA filter without issue [Ingress] [Opa]", func() {
+		jig = ingress.NewIngressTestJig(f.ClientSet)
+		cs = f.ClientSet
+		serviceName := "skipper-ingress-test"
+		ns := f.Namespace.Name
+		hostName := fmt.Sprintf("%s-%d.%s", serviceName, time.Now().UTC().Unix(), E2EHostedZone())
+		labels := map[string]string{
+			"app": serviceName,
```

Comment:
I remember when testing OPA that the application name should be equal policy name without `-policiy`, is this true?

Maybe it's better to name resources for this as `styra-smoketest` and use it elsewhere.

Reply:
`the application name should be equal policy name without -policiy, is this true?` this is more like a convention. When writing policies for a certain application, we name the policy repo with the template `<application-name>-policies`